apps/dot-voting/app/store/init.js

```diff
@@ -30,10 +29,12 @@ export const initStore = addressBookAddress => {
       // always return the state even unmodified
       return state
     },
-    [
-      of({ event: INITIALIZATION_TRIGGER }),
-      // handle address book events
-      addressBookApp.events(),
-    ]
+    {
+      externals: [
+        {
+          contract: addressBookApp
```

Comment:
I understand that we want to keep the working state in this branch, but this is extra work and not very efficient, this shouldn't be required since we are getting rid of the address-book integration in another PR that will be merged in hours/days